src/components/StartPage.vue

```diff
@@ -19,23 +19,20 @@
         type="number"
         placeholder="Question amount"
         min="1"
-        :max="questions.length"
       />
     </div>
     <button class="start-button" @click="startQuiz">Start</button>
   </div>
 </template>
 
 <script setup lang="ts">
-import { MCQQuiz } from "@/types/MCQ";
 import { ref } from "vue";
-const { questions } = defineProps<MCQQuiz>();
 const questionAmount = ref<number>(0);
-const tag = ref<string>("");
+const tag = ref<string | null>(null);
```

Comment:
this is "Tag" but the value actually contains multiple tags. Maybe replace it for "tags"?